base/record_replay.h

```diff
@@ -192,6 +274,42 @@ class SCOPED_LOCKABLE AutoUnlockMaybeEventsDisallowed {
   base::Lock& lock_;
 };
 
+
+
+class SCOPED_LOCKABLE ReplayAutoLock {
```

Comment:
Similarly, do we need this class if we don't have a replay specific locking class?